### ISSUE TITLE
Referring to this project by name

### ISSUE THREAD

**Issue by lucascolley** [data-apis/array-api#759]
This project is referred to using a variety of names, including:

1. (the) Python array API standard
2. (the) array API standard
3. (the) Array API standard
4. (the) Array API

As far as I can tell, (1) is the closest we have to an official name for the project, as the title of the docs.

"Python" makes sense to drop in a lot of contexts. For example (2) is used in the description of this repo and throughout NEP 56. I usually use (2).

"Array" is often capitalised. For example, the Quansight Labs Annual Report 2023 uses (3) throughout. Maybe this is more grammatically correct, I'm not sure.

"Standard" is often dropped. I have seen (4) a lot in discussions on consumer repos. I think that this has the potential to be confusing in some scenarios where other array APIs are also being discussed.

---

Maybe this is just bike-shedding to the extreme, but I think it would be good to improve the consistency of these references. I would suggest that we try to always use (2), with (1) where the context of Python isn't obvious, and the wider community should catch on. Such a well-thought-out API standard deserves a standard name :)

**Comment by izaid** [comment]
Weirdly, this has been on my mind as well whenever I write something about (the) array API standard. At some point, I mentally chose to go with 2).

**Comment by rgommers** [comment]
Yes, it's a little tricky indeed. I have just cleaned up all the NumPy 2.0 release notes to use (2). I'd say that (1) is the official name, and if it's clear we're within the Python world, then I think using (2) is fine.

I'd like to avoid (3) as an unnecessary deviation of capitalization, and (4) because it is too generic.
